lib/jsonapi_suite/strong_resources/exception_handler.rb

```diff
@@ -0,0 +1,37 @@
+module JsonapiSuite
+  module StrongResources
+    class ExceptionHandler < JsonapiErrorable::ExceptionHandler
```

Comment:
Can this class live within SR itself? I am thinking it might be more testable/maintainable long-term.

Reply:
I actually didn't know we were the owners of `SR`, which is why I did it here. That makes much more sense yes.